index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav id="navbar">
+        <div class="container">
+          <a href="#" class="page_logo">My Logo</a>
+          <div class="navbar-nav">
+            <button type="button" class="nav-toggle-btn">
+              <img
+                src="./assets/images/icons/Union@2x.png"
+                width="3rem"
+                alt="menu-bar"
+              />
+            </button>
+          </div>
+        </div>
+      </nav>
+      <section id="headline">
+        <div class="container">
+          <h1 class="headline-title">
+            I’m Mariane <br />
+            Glad to see <br />
+            you!
```

Comment:
- [ ] Please make sure that the font color in the title and the Paragraph matches the one from  Figma design

| Your code | Figma |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/211920610-9d2f44c2-5d4f-41a5-86d0-1928e67415b2.png) | ![image](https://user-images.githubusercontent.com/74744787/211920691-08db33c0-cfe2-42a8-b499-f51817d6fa86.png)  |

Reply:
Hello @MeqdamAlqudah 👋

I'm already using the same font color as the one indicated in figma. I saved it in a variable and referenced it in the color attribute of the title case

![image](https://user-images.githubusercontent.com/42853016/211998078-abf86aa7-79e0-442b-b2fe-7f4ff92e5535.png)

![image](https://user-images.githubusercontent.com/42853016/211998132-f619a011-91d6-4819-9236-90ef1e7d148e.png)

*The .page_logo is the class name for the logo title*